docs/src/content/docs/ref/core-strings.mdx

```diff
@@ -165,6 +165,28 @@ let buzz: Slice = s.toCell().asSlice();
 fizz == buzz; // true
 ```
 
+## String.hashData
+
+<Badge text="Available since Tact 1.6 (not released yet)" variant="tip" size="medium"/><p/>
+
+```tact
+extends fun hashData(self: String): Int;
+```
+
+Extension function for the [`String{:tact}`][p] type.
+
+Calculates and returns an [`Int{:tact}`][int] value of the [SHA-256][sha-2] hash of the data bits from the given [`String{:tact}`][p], which should have a number of bits divisible by $8$.
+
+Unlike [`sha256(){:tact}`](/ref/core-math#sha256), this function is gas-efficient and **only** hashes up to $127$ bytes of the given [`String{:tact}`][p]. Using longer strings would cause collisions if their first $127$ bytes are the same.
```

Comment:
let's not over-use links here (`String` is already linked in the previous paragraphs).
```suggestion
Unlike [`sha256(){:tact}`](/ref/core-math#sha256), this function is gas-efficient and **only** hashes up to $127$ bytes of the given string. Using longer strings would cause collisions if their first $127$ bytes are the same.
```